WordPress/WordPressStatsWidgets/Views/Localization/LocalizableStrings.swift

```diff
@@ -36,10 +36,18 @@ enum LocalizableStrings {
                                                    defaultValue: "Best views ever",
                                                    comment: "Title of best views ever label in all time widget")
     // Unconfigured view
-    static let unconfiguredViewTitle = LocalizedStringKey("widget.today.unconfigured.view.title",
+    static let unconfiguredViewTodayTitle = LocalizedStringKey("widget.today.unconfigured.view.title",
                                                           defaultValue: "Log in to WordPress to see today's stats.",
                                                           comment: "Title of the unconfigured view in today widget")
 
+    static let unconfiguredViewAllTimeTitle = LocalizedStringKey("widget.alltime.unconfigured.view.title",
+                                                          defaultValue: "Log in to WordPress to see all time stats.",
+                                                          comment: "Title of the unconfigured view in all time widget")
+
+    static let unconfiguredViewThisWeekTitle = LocalizedStringKey("widget.thisweek.unconfigured.view.title",
+                                                          defaultValue: "Log in to WordPress to see this week stats.",
```

Comment:
I would probably change this one to "this week's stats". The all time one is fine, though.